components/eamxx/src/physics/share/tests/physics_saturation_run_and_cmp.cpp

```diff
@@ -50,15 +50,17 @@ struct UnitWrap::UnitTest<D>::TestSaturation
     sat_ice_fp  = physics::polysvp1(temps, true, Smask(true))[0];
     sat_liq_fp  = physics::polysvp1(temps, false, Smask(true))[0];
 
-    mix_ice_fr = physics::qv_sat(temps, pres, true, Smask(true), physics::Polysvp1)[0];
-    mix_liq_fr = physics::qv_sat(temps, pres, false,Smask(true), physics::Polysvp1)[0];
+    //Functions<S,D>::qv_sat_dry(const Spack& t_atm, const Spack& p_atm_dry, const bool ice, const Smask& range_mask, 
```

Comment:
Unless we completely delete qv_sat, we should probably continue testing it here... so *add* new tests instead of just replacing the qv_sat tests with qv_sat_dry tests.